docs/README.md

```diff
@@ -1,13 +1,14 @@
 
 # Design Docs
+KFServing architecture consists of two main specifications, around control pland and data plane. Control plane is responsible for deploying and managing the lifecycle of models, while data plane is responsible for defining the contracts to interact with models once deployed.
```

Comment:
KFServing's
pland -> plane

Reply:
ah - should do a spell check next time :-)